Makefile

```diff
@@ -24,6 +24,8 @@ INTEGRATION_TEST_DIR := ./test/e2e
 GO-FUZZ-BUILD := $(TOOLS_BIN_DIR)/go-fuzz-build
 GENSRC = pkg/generated/models/%.go pkg/generated/client/%.go
 SRCS = $(shell find pkg -iname "*.go"|grep -v pkg/generated) $(GENSRC)
+TEST_COVERAGE_PERCENTAGE = 70
```

Comment:
I have set the global test coverage to be `70%`. I didn't know what to start with. Open to what the project wants it to be.